crates/crates-io/Cargo.toml

```diff
@@ -15,9 +15,9 @@ path = "lib.rs"
 
 [dependencies]
 curl = "0.4"
-failure = "0.1.1"
+failure = { version = "0.1.5", default_features = false }
 http = "0.1"
 serde = { version = "1.0", features = ['derive'] }
 serde_derive = "1.0"
 serde_json = "1.0"
-url = "1.0"
+url = "1.1"
```

Comment:
Is there a reason to make this change?  I think we're currently using 1.7.  We're a little lax about keeping these in sync, though we probably should.